src/patternfly/components/Table/table-th.hbs

```diff
@@ -12,7 +17,7 @@
     {{{table-th--attribute}}}
   {{/if}}>
   {{#if table-th--sortable}}
-    <button>
+    {{#> button}}
```

Comment:
Do you mean to use the button component here? If so you need to use it with a modifier.

Reply:
updated, thanks!